Dockerfile

```diff
@@ -0,0 +1,6 @@
+
+FROM openjdk:17-slim
+
+ADD ./target/chat-dancer.jar /chat-dancer.jar
+CMD ["java",  "-jar", "/chat-dancer.jar"]
+EXPOSE 8080
```

Comment:
In order to run the .jar file in my machine I have to use the SNAPSHOT.jar
Should I include that here also?